library-no-op/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -11,21 +11,25 @@ object Chucker {
     const val SCREEN_HTTP = 1
     const val SCREEN_ERROR = 2
 
-    val isOp = false
+    const val isOp = false
```

Comment:
I'm afraid you have to revert this line (unless we come up with a different solution).
I just realized that `const val` properties won't generate getters/setters in the Bytecode. This will result in a breaking change on the Java side of things.

tl;dr: Java users accessing `Chucker.isOp()` will have a compilation issue as now it should be accessed as `Chucker.isOp()`

We should consider maybe leaving a note over here and ship this change with Chucker 4.x

Reply:
Fixed it. The possibility is quite low, but yes, better safe than sorry.

Reply:
@cortinico There was a typo, probably in your tl;dr. Could you fix it, since you referenced this comment in the library code and we shouldn't confuse anyone.

Reply:
Yup thanks for noticing it